### PR TITLE
Setup linters

### PR DESCRIPTION
**Pull request by lewisjn-arch**
#### This PR has:

- [ ] First Project setup
- [ ] Basic html and css
- [ ] Linters installation and git flow.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,178 +24,93 @@ For more information on the importance of a professional README for your reposit
 
 <!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [First-Project] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**[First-Project]** is a html/css project that I have done to teach me how to use linters.
```
Comment (AlphaNtihinduka) [inline README.md]: - [x] Kindly consider providing a descriptive name title of the project in the readme. This is to ensure that the readme is professional and repository visitor can have a quick understanding of what the project is about. At most cases, the title may be same as the repository name ☺️.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @lewisjn-arch 🙋‍♂️
Good job working on your hello Microverse project. You have done incredibly well. 👏  

You still need to implement few on the project but you are almost there!!! 🚶: 

TO HIGHLIGHT:
- Your project is professional ✅
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

- _Kindly consider the inline review comments ☺️._

## Optional suggestions
- [ ] [OPTIONAL] Kindly consider writing the commit messages starting with Uppercase 👍.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

File: README.md (hunk 1 above)
Comment (lewisjn-arch) [reply README.md]: Thank you, I will edit that.

**Review by osamaashraf6 — APPROVED** [review]
Hi @lewisjn-arch 😊, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
MinionsGIF (3)

## Highlights🎉🎉

 
- [ ] Descriptive pull request💯
- [ ] Professional README file👍
- [ ] Good adding the test file and node modules at gitignore👌

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**